trunk/SUAVE/Components/Energy/Networks/Ramjet.py

```diff
@@ -0,0 +1,243 @@
+        thrust                    = self.thrust
+        number_of_engines         = self.number_of_engines
+
+        #Creating the network by manually linking the different components
+
+        #set the working fluid to determine the fluid properties
+        ram.inputs.working_fluid                               = self.working_fluid
+
+        #Flow through the ram , this computes the necessary flow quantities and stores it into conditions
+        ram(conditions)
+
+        #link inlet nozzle to ram
+        inlet_nozzle.inputs.stagnation_temperature             = ram.outputs.stagnation_temperature
+        inlet_nozzle.inputs.stagnation_pressure                = ram.outputs.stagnation_pressure
+
+        #Flow through the inlet nozzle
+        inlet_nozzle(conditions)
+
+        #link the combustor to the inlet nozzle
+        combustor.inputs.stagnation_temperature                = inlet_nozzle.outputs.stagnation_temperature
+        combustor.inputs.stagnation_pressure                   = inlet_nozzle.outputs.stagnation_pressure
+        combustor.inputs.mach_number                           = inlet_nozzle.outputs.mach_number
+
+        #flow through the high pressor comprresor
+        combustor.compute_rayleigh(conditions)
+
+
+        #link the core nozzle to the low pressure turbine
+        core_nozzle.inputs.stagnation_temperature              = combustor.outputs.stagnation_temperature
+        core_nozzle.inputs.stagnation_pressure                 = combustor.outputs.stagnation_pressure
+
+        #flow through the core nozzle
+        core_nozzle.compute_limited_geometry(conditions)
+
+        # compute the thrust using the thrust component
+        #link the thrust component to the core nozzle
+        thrust.inputs.core_exit_velocity                       = core_nozzle.outputs.velocity
+        thrust.inputs.core_area_ratio                          = core_nozzle.outputs.area_ratio
+        thrust.inputs.core_nozzle                              = core_nozzle.outputs
+
+        #link the thrust component to the combustor
+        thrust.inputs.fuel_to_air_ratio                        = combustor.outputs.fuel_to_air_ratio
+
+        #link the thrust component to the low pressure compressor
+        thrust.inputs.stag_temp_lpt_exit                       = core_nozzle.outputs.stagnation_temperature
+        thrust.inputs.stag_press_lpt_exit                      = core_nozzle.outputs.stagnation_pressure
+        thrust.inputs.number_of_engines                        = number_of_engines
+        thrust.inputs.flow_through_core                        =  1.0 #scaled constant to turn on core thrust computation
+        thrust.inputs.flow_through_fan                         =  0.0 #scaled constant to turn on fan thrust computation
+
+        #compute the thrust
+        thrust(conditions)
+
+        #getting the network outputs from the thrust outputs
+        F            = thrust.outputs.thrust*[1,0,0]
```

Comment:
This has an angle assumption that should be adjusted. Come talk to me about this. We want to be able to angle propulsors from now on

Reply:
can you link to an example, to keep consistent

Reply:
https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Components/Energy/Networks/Battery_Propeller.py#L175